wheatley/bot.py

```diff
@@ -137,7 +138,15 @@ def _on_row_gen_change(self, row_gen_json: JSON) -> None:
 
     def _on_size_change(self) -> None:
         self._check_number_of_bells()
-        self._rounds = rounds(self.number_of_bells)
+        self._check_starting_row()
+        self._rounds = generate_starting_row(self.number_of_bells, self.row_generator.custom_start_row)
```

Comment:
I think these are the wrong way round - we should check the newly created starting row?